Lib/test/test_os.py

```diff
@@ -102,7 +102,12 @@ def test_getcwd_long_path(self):
         # On Windows, the test can stop when trying to create a path longer
         # than MAX_PATH if long paths support is disabled:
         # see RtlAreLongPathsEnabled().
-        min_len = 2000   # characters
+        #
+        # On VxWorks, PATH_MAX is defined as 1024 bytes.
+        if sys.platform == 'vxworks':
+            min_len = 1000
+        else:
```

Comment:
why not just:
```
        min_len = 2000   # characters
        #
        # On VxWorks, PATH_MAX is defined as 1024 bytes.
        if sys.platform == 'vxworks':
            min_len = 1000
```

also I am confused with the PATH_MAX statement when defining min_len

Reply:
Changed as you proposed. VxWorks RTOS defines PATH_MAX as 1024. Making a path with longer name than PATH_MAX will fail. That is, os.mkdir() will fail in this test case. I also add more explain in the code comments.